source/common/network/connection_impl.cc

```diff
@@ -44,6 +44,17 @@ std::ostream& operator<<(std::ostream& os, Connection::State connection_state) {
   return os;
 }
 
+absl::string_view ipVersionAsString(Network::Address::IpVersion ip_version) {
+  switch (ip_version) {
```

Comment:
if we're not worried about unknown you could
return ip_version == Network::Address::IpVersion::v4 ? "v4" : "v6"
If we think we could end up with unknown please make sure it's unit tested?

Reply:
I don't think it's possible to get into the default case and a lot of code will be broken anyway. I updated the code to use your suggestion.